web/src/components/feature/calculation_input/CalculationInput.tsx

```diff
@@ -164,6 +165,7 @@ export const CalculationInput = ({
           componentProperties={componentProperties}
           editablePackage={selectedPackage}
           savePackage={(x) => savePackage('edit', x)}
+          packages={packages}
```

Comment:
Does it make sense to be able to select a template when editing?

Reply:
Was thinking about that as well. It might be handy in some cases (maybe?). And it doesn't really hurt.